contracts/Comptroller.sol

```diff
@@ -142,31 +186,27 @@ contract Comptroller is
      * @dev Sender must not have an outstanding borrow balance in the asset,
      *  or be providing necessary collateral for an outstanding borrow.
      * @param vTokenAddress The address of the asset to be removed
-     * @return Whether or not the account successfully exited the market
+     * @return error Always NO_ERROR for compatibility with Venus core tooling
```

Comment:
add emitted event to natspec

Reply:
Can we add this note as the dev nat spec too because that is the one we will look for in our automated docs

Reply:
What tooling are we maintaining compatibility for?